### PR TITLE
ismecha define

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

**Comment by Cyberboss** [comment]
Travis

File: code/game/objects/items/storage/storage.dm [hunk 1]
```diff
@@ -34,7 +34,7 @@
 		if(!over_object)
 			return
 
-		if (istype(usr.loc, /obj/mecha)) // stops inventory actions in a mech
+		if (ismecha(M.loc)) // stops inventory actions in a mech
```
Comment (nicbn) [inline code/game/objects/items/storage/storage.dm]: ismecha(usr.loc)

File: code/modules/clothing/clothing.dm [hunk 2]
```diff
@@ -48,7 +48,7 @@
 	if(pockets && over_object == M)
 		return pockets.MouseDrop(over_object)
 
-	if(istype(usr.loc, /obj/mecha)) // stops inventory actions in a mech
+	if(ismecha(M.loc)) // stops inventory actions in a mech
```
Comment (nicbn) [inline code/modules/clothing/clothing.dm]: ismecha(usr.loc)

File: code/game/objects/items/storage/storage.dm (hunk 1 above)
Comment (KorPhaeron) [reply code/game/objects/items/storage/storage.dm]: Look a few lines up

**Pull request merged**